server/db/migrations/20230309195953_update-campaign-model.js

```diff
@@ -0,0 +1,20 @@
+const tableName = 'campaigns'
+
+module.exports = {
+  async up(knex) {
+    await knex.schema.alterTable(tableName, (table) => {
+      // Auto-incrementing non-nullable unsigned integer primary key "id" field
+      table.increments()
```

Comment:
If you are altering the table, there is no need to add `increments()` again. It will cause a duplicate key error.